webapp/channels/build/emoji/make_emojis.mjs

```diff
@@ -13,17 +13,17 @@
 * npm run make-emojis -- --help
 */
 
-import path from 'node:path';
-import * as fsPromise from 'node:fs/promises';
 import * as fs from 'node:fs';
+import * as fsPromise from 'node:fs/promises';
+import path from 'node:path';
 import * as url from 'node:url';
 
-import yargs from 'yargs';
 import chalk from 'chalk';
-import jsonData from 'emoji-datasource/emoji.json';
-import jsonCategories from 'emoji-datasource/categories.json';
+import jsonCategories from 'emoji-datasource/categories.json' assert {type: 'json'};
+import jsonData from 'emoji-datasource/emoji.json' assert {type: 'json'};
+import yargs from 'yargs';
 
-import additionalShortnames from './additional_shortnames.json';
+import additionalShortnames from './additional_shortnames.json' assert {type: 'json'};
```

Comment:
Question : Why is assert needed?

Reply:
Node.js needs it to be able to import json files now. We don't need this in code that's actually part of the web app though because Webpack doesn't have that same requirement